app/models/vm_or_template/operations/relocation.rb

```diff
@@ -23,7 +24,7 @@ def evacuate(options = {})
   end
 
   def validate_evacuate
```

Comment:
I kept the method definition as it is and changed method body to internally use supports_evacuate, so that if old method is called from any other place, the code still works. But we can remove it as the travis build is successful.